app/controllers/patients_controller.rb

```diff
@@ -51,6 +51,7 @@ def download
                                     params[:case_manager_name].to_s)
                                .generate_pledge_pdf
       @patient.update pledge_generated_at: Time.zone.now
+      @patient.update pledge_generated_by: current_user.id
```

Comment:
We can save using an object here, it'll make it a little more readable (get them small gains!!). Go ahead and take off the `id` here and just save `current_user` and that should blend okay.

Could you also add this update to the previous one, so it updates both `pledge_generated_at` and `pledge_generated_by` at the same time? No need to do two db calls when we can do one!